types/chai-uuid/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for chai-uuid 1.0
+// Project: https://github.com/rfrench/chai-uuid
+// Definitions by: Harm van der Werf <https://github.com/harm-less>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
+/// <reference types="chai" />
+
+declare global {
+	namespace Chai {
+		type UuidVersion = 'v1' | 'v2' | 'v3' | 'v4' | 'v5' | '';
+
+		interface Assertion extends LanguageChains, NumericComparison, TypeComparison {
+			uuid(uuid?: UuidVersion, msg?: string): void;
+			guid(guid?: any, msg?: string): void;
+		}
+
+		interface Assert {
+			uuid(uuid: string, version?: UuidVersion, msg?: string): void;
+			guid(guid: string, version?: any, msg?: string): void;
+		}
+	}
+}
+
+declare function chaiUuid(chai: any, utils: any): void;
+declare namespace chaiUuid {}
```

Comment:
It seems the empty namespace can be removed, can you confirm?

Reply:
Hello @plantain-00 ,

Thanks for taking the time to review these definitions.
But as far as I know, they do need to be there (partly). If I remove it gives me this error:
![image](https://user-images.githubusercontent.com/6368074/39129452-d2444c84-470a-11e8-87c5-2bf978f87b53.png)

So it seems I've to export something even though I'm not exactly sure why this is. 
I was inspired by this definition just so you know how I came to this in the first place: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chai-json-schema/index.d.ts#L27-L29 

I removed the empty namespace and removed a parameter that apparently wasn't implemented in the plugin and it seems to be working just fine!

While I was writing this message, a few new things about typings dawned on me, so that was nice 😄